setup.py

```diff
@@ -9,26 +9,23 @@
 README = (HERE / "README.md").read_text()
 
 setup(
-    name = "osc-ingest-tools",
```

Comment:
is this some kind of opinionated code formatting check?

Reply:
Yeah, 'black' is a widely used and very opinionated automated formatter but is super easy to use and, since devs are liable to form their own conventions, is a big help with making code bases consistent.